packages/@react-spectrum/tooltip/test/TooltipTrigger.test.js

```diff
@@ -55,7 +51,7 @@ describe('TooltipTrigger', function () {
 
   describe('click related tests', function () {
 
-    it('triggered by click event', async function () {
+    it('a click event can open the tooltip', async function () {
```

Comment:
what's the difference between this test and the one above it?

Reply:
In the first one I am checking Id's (line 59 and and 60). In the one you referenced here I am not

Reply:
Right, so why do we need this test, what is the value that this test adds over the previous one?